independent-projects/resteasy-reactive/pom.xml

```diff
@@ -144,6 +158,12 @@
                 <version>${jandex.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>net.bytebuddy</groupId>
+                <artifactId>byte-buddy</artifactId>
+                <version>${bytebuddy.version}</version>
```

Comment:
Given that it's a managed dependency, the version could be removed.

Reply:
Fixed

Reply:
I made it managed, so I removed the version :)